### ISSUE TITLE
Missing links in core example index for sendfd.py and recvfd.py examplesd

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5657]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5657|
|Type|defect|
|Created|2012-04-26 15:31:59Z|

These examples are mentioned in the index, but the links for them go nowhere.

Attachments:

* [sendfd.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/829/829f88fd420e51a15b0da7dd236791253f17941c/b01b75134d9ab4def6de57d64b582947a750ccd3.patch) (524 bytes) - added by exarkun on 2012-04-26 15:33:35Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5657 5657
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1335454319000000 1335454319000000
changetime__1335454714000000 1335454714000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_2"></a>|@itamarst set owner to @exarkun|
|-|-|

Looks good, merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [34296]) Add proper link targets to new sendFileDescriptor entries in the core example index

Author: exarkun
Reviewer: itamar
Fixes: [#5657](https://github.com/twisted/twisted/issues/5657)
